DataFormats/Portable/interface/PortableHostCollection.h

```diff
@@ -94,6 +94,8 @@ class PortableHostCollection {
     layout.ROOTStreamerCleaner();
   }
 
+  void aggregate(ConstView& view) { layout_.aggregate(view); }
```

Comment:
This new member function would benefit from documentation.

Can the signature be
```suggestion
  void aggregate(ConstView const& view) { layout_.aggregate(view); }
```
?

Reply:
@makortel just to confirm, is the name of the new function OK for you ?

Reply:
I don't object, but I personally don't find the name very clear (I'd have to resort to reading documentation). Is the purpose to `memcpy()` the contents of the columns and scalars of the argument `view` into the buffer held by `PortableHostCollection`? In that case, how about something like `memcpyFrom(ConstView const& view)`? Or a constructor overload? (I'm not sure if assignment operator would be clear or confusing)